### PR TITLE
Fixing typo Changing planing to planning

### PR DESCRIPTION
**Pull request by Jberma23**
**Pull Request Requirements:**

* Provide a brief description of the changeset.
* Include a screenshots if applicable
* Ensure that the changeset adheres to the [DCO guidelines](https://github.com/apps/dco).

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjE0NWJiMzI0N2IzZDJhZDI5MjI0N2UiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.Bz7SrRZaWbDKmWWX5WdebDg_VgtMhkrX7j1GCfbS5Mk">Huly&reg;: <b>UBERF-6432</b></a></sub>

### CODE REVIEW

**Comment by aonnikov** [comment]
@Jberma23 could you please sign-off your commit?

**Comment by Jberma23** [comment]
@aonnikov sorry about that so used to it auto signing off on my work computer forgot it doesn't on my personal one. Should be fixed now

File: packages/theme/styles/love.scss [hunk 1]
```diff
@@ -131,3 +131,30 @@
     .floorGrid-room__field { visibility: hidden; }
   }
 }
+
+// RoomPopup
```
Comment (aonnikov) [inline packages/theme/styles/love.scss]: These changes does not seem to be related to the typo fix. Could you remove them?

Comment (Jberma23) [reply packages/theme/styles/love.scss]: yeah removed. Not sure where that came from. Sorry about that

**Review by aonnikov — APPROVED** [review]
